src/button/README.md

```diff
@@ -2,7 +2,48 @@
 
 The `vscode-button` is a web component implementation of a [button element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button). The `vscode-button` also supports several visual appearances––primary, secondary, and icon.
 
-## Attributes
+![Button hero](/docs/assets/button-hero.png)
+
+## Usage
+
+### Types
+
+| Type      | Example                                               | Usage                                                                              |
+| --------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| Primary   | ![Primary](/docs/assets/button-types-primary.png)     | Emphasizes the highest priority action in a view.                                  |
+| Secondary | ![Secondary](/docs/assets/button-types-secondary.png) | Used for additional actions in a view that already features a primary action.      |
+| Icon      | ![Icon](/docs/assets/button-types-icon.png)           | A space-efficient style that renders a single icon to represent a specific action. |
```

Comment:
Could we experiment with making these images smaller? They're taking up a lot of space in the table imo

Reply:
Yep—that's on my todo list! Need to fix resolutions across the PR

Reply:
Fixed by switching to img tags—thanks for the tip!